test/test_core.py

```diff
@@ -9119,8 +9119,13 @@ def test_pthread_create(self):
     self.do_run_in_out_file_test('core/pthread/create.cpp')
 
   @node_pthreads
-  def test_pthread_c11_threads(self):
-    self.set_setting('PROXY_TO_PTHREAD')
+  @parameterized({
+    'unpooled': ([],),
+    'pooled': (['-sPTHREAD_POOL_SIZE=1'],),
+    'proxied': (['-sPROXY_TO_PTHREAD'],),
+  })
+  def test_pthread_c11_threads(self, args):
+    self.emcc_args += args
     self.set_setting('EXIT_RUNTIME')
```

Comment:
Can `EXIT_RUNTIME` be removed here?

Reply:
For non-proxied case it could be, but for PROXY_TO_PTHREAD it's still necessary. Do you want me to move it into relevant option array?

Reply:
> I'll try to move it anyway; I'm now curious whether it'll work for non-proxied case.
> 
> >

If you have any other comments meanwhile though (while waiting for CI), please let me know.

Reply:
> I'll try to move it anyway; I'm now curious whether it'll work for non-proxied case.

Yeah it does work, so I'll keep it only for the PROXY_TO_PTHREAD case.